test/parallel/test-child-process-stdio-big-write-end.js

```diff
@@ -29,7 +30,7 @@ function parent() {
 
   // Write until the buffer fills up.
   do {
-    var buf = new Buffer(BUFSIZE);
+    buf = new Buffer(BUFSIZE);
     buf.fill('.');
     sent += BUFSIZE;
   } while (child.stdin.write(buf));
```

Comment:
@silverwind Yeah, I thought we can do that cleanup separately with a separate PR with the `no-redeclare` rule.